config/routes.rb

```diff
@@ -1,5 +1,5 @@
 if ShopifyApp.configuration.routes_enabled?
-  ShopifyApp::Engine.routes.draw do
+  Rails.application.routes.draw do
```

Comment:
I kinda prefer to be explicit then just magically have the session routes working once you install the gem.
If we want to be minimalistic maybe have something similar to Devise and have a method that can be called in the `routes.rb` to define the routes?